src/App.jsx

```diff
@@ -1,10 +1,9 @@
 import axios from 'axios';
 import 'bootstrap-dark-5/dist/css/bootstrap-nightshade.min.css';
```

Comment:
do we still need that?

Reply:
It is still used for the CSRF token - line 35. Bootstrap itself is used within the wohle UI.

Reply:
bootstrap-dark has been removed